src/page_body/structure/blocks/page_block_figma_components.pr

```diff
@@ -1 +1,104 @@
+
+{[ if (variant === "Grid") ]}
+    <div class="figma-components" style="grid-template-columns: repeat({{ numberOfColumns }}, 1fr);">
+        {[ for component in components ]}
+            {[ if (isComponentSelected(block.selectedFigmaComponents, component.id)) ]}
+                {[ const selectedPropertiesIds = block.selectedFigmaComponents[0].selectedComponentProperties /]}
+                {[ const componentPreviews = getComponentPreviews(component, selectedPropertiesIds, block.selectedFigmaComponents.length > 1) /]}
+                {[ for componentPreview in componentPreviews ]}
+                    <div class="figma-component">
+                        <div class="figma-component-image figma-component-image-border figma-component-image-background" style="{{ block.figmaComponentsBlockConfig.previewContainerSize === "Centered" ? "height: 116px;" : "" }}">
+                            <img src="{{ componentPreview.component.thumbnailUrl }}" alt="{{ componentPreview.name }}" height="{{ componentPreview.component.origin?.height }}" width="{{ componentPreview.component.origin?.width }}" />
+                        </div>
+                        <div class="figma-component-details">
+                            {[ if (block.figmaComponentsBlockConfig.showComponentName) ]}
+                                <div class="figma-component-name">
+                                    {{ componentPreview.name }}
+                                </div>
+                            {[/]}
+                            {[ if (block.figmaComponentsBlockConfig.showPropertyList && componentPreview.properties.length > 0) ]}
+                                <div class="figma-component-properties">
+                                    {[ for property in componentPreview.properties ]}
+                                        <span class="figma-component-property-name">
+                                            {{ property.name }}
+                                        </span>
+                                        <span class="figma-component-property-value">
+                                            {{ property.value }}
+                                        </span>
+                                    {[/]}
+                                </div>
+                            {[/]}
+                            {[ if (block.figmaComponentsBlockConfig.showComponentDescription) ]}
+                                <div class="figma-component-description">
+                                    {{ componentPreview.component.description }}
+                                </div>
+                            {[/]}
+                        </div>
+                    </div>
+                {[/]}
+            {[/]}
+        {[/]}
+    </div>
+{[/]}
+
+{[ if (variant === "Canvas") ]}
+    <div class="figma-components figma-components-canvas figma-component-image-background" style="grid-template-columns: repeat({{ numberOfColumns }}, 1fr);">
```

Comment:
Same as above for naming conventions for classes/ID

Reply:
Note: It will need to change also in `page_block_figma_component_props_table.pr`, there is now empty `<div>`

Reply:
[6dbd463](https://github.com/Supernova-Studio/exporter-documentation/pull/167/commits/6dbd46389004133fc1529e0db4877b3fa7456de8)